index.html

```diff
@@ -90,38 +91,45 @@ <h6 class="article__title">
 									</a>
 								</p>
 							</article>
+						</section>
 
 					</div>
 				</div>
 
 				<div class="sidebar col-lg-4">
 						<h2 class="heading-underline">Upcoming events</h2>
-						<article class="article">
+						<section class="fundraisers1">
 							<a href="#">
```

Comment:
Add a class to all the `<a>` tags that contain an image to size the images inside better. Use `width: 30%' and then on the text next to it do 'width: 70%' so that all the images line up at 30% across each section and all the text boxes line up too.